backrest/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  backrest:
+    image: garethgeorge/backrest:latest
+    volumes:
+      - ./backrest/data:/data
+      - ./backrest/config:/config
+      - ./backrest/cache:/cache
+      - backrest-restores:/backrest-restores
```

Comment:
Shouldn't these only be in the docker-compose.instance.yaml and use BACKREST_USE_EXTERNAL_VOLUME and BACKREST_EXTERNAL_VOLUME vars?

Reply:
The `backrest-restores` volume is defined in `docker-compose.instance.yaml` as either a named volume or a bind mount, based on `BACKREST_USE_EXTERNAL_VOLUME`.

I realize the other 3 volumes were bind mounts so I changed them in [0ac8e3b](https://github.com/EnigmaCurry/d.rymcg.tech/pull/363/commits/0ac8e3bdb2cb3f2a11c8e6bce1f6ccf3ccca4f96) to named volumes.